sos/report/plugins/rpm.py

```diff
@@ -19,7 +19,9 @@ class Rpm(Plugin, RedHatPlugin):
     option_list = [("rpmq", "queries for package information via rpm -q",
                     "fast", True),
                    ("rpmva", "runs a verify on all packages", "slow", False),
-                   ("rpmdb", "collect /var/lib/rpm", "slow", False)]
+                   ("rpmdb", "collect /var/lib/rpm", "slow", False),
+                   ("rpmshowrc", "shows the options in rpmrc and macros files",
+                    "fast", True)]
```

Comment:
Does this really need to be gated behind a plugin option? We usually reserve those options for long running items, potentially sensitive collections, or historically ones that could make changes to the system (before the addition of `--allow-system-changes` and the like).

From quick testing this looks like it should be safe for a default collection.

Reply:
I do agree that the extra option (though enabled by default) is bit redundant - I cant figure out a use case when one would need to prevent collecting the cmdoutput.

Reply:
I chose this approach to make sure that it was consistent with the other options, but you are right, so let me re-send this PR.